index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="index.css" />
+  <title>Awesome Books</title>
+</head>
+<body>
+<main>
+  <header>
+    <nav class="nav">
+      <a href="">Awesome Books</a>
+      <ul class="nav__links">
+        <li id="nav__links__list"> <a href="#">List</a></li>
+        <li id="nav__links__add"> <a href="#">Add new</a></li>
+        <li id="nav__links__contacts"> <a href="#">Contacts</a></li>
+      </ul>
+    </nav>
+  </header>
+  <section class="book">
+    <p class="date-time" id='date-time'></p>
+    <h1>
+      All awesome books
+    </h1>
+    <table class="book-list">
+
+    </table>
+    <hr>
+    <form action="#" id="form" class="form">
+      <h2>Add a new book</h2>
+      <input type="text" name="title" class="title" id="title" placeholder="Title" required /> <br>
+      <input type="text" name="author" class="author" id="author" placeholder="Author" required /> <br>
+      <button type="submit">Add</button>
```

Comment:
- [ ] [OPTIONAL] It would be nice if you added a link such that when a user submits the book, the user gets redirected to the collection. It would be a nice addition to an already good project